### PR TITLE
Transactions mobile

### LINKED ISSUE #595: Implement Transactions mobile screen
https://invis.io/GZNS9NQCYV3

### PR DESCRIPTION
**Pull request by valentinewallace**
Closes #595.

### CODE REVIEW

File: src/component/list.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ export class List extends Component {
         ref={component => (this.list = component)}
         dataSource={this.dataSource}
         renderHeader={this.props.renderHeader}
-        stickyHeaderIndices={[0]}
+        stickyHeaderIndices={this.props.renderHeader ? [0] : []}
```
Comment (tanx) [inline src/component/list.js]: Why is this needed?

Comment (valentinewallace) [reply src/component/list.js]: Otherwise, if you have a list with no header then the first item of the list sticks to the top of the page on scroll.

File: src/view/transaction-mobile.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+});
+
+const TransactionListItem = ({ tx, unitLabel, onSelect }) => (
+  <ListItem style={iStyles.item} onSelect={onSelect}>
+    <View
+      style={[
+        iStyles.icon,
+        {
+          backgroundColor:
+            tx.type === 'lightning' ? color.purple : color.orange,
```
Comment (valentinewallace) [inline src/view/transaction-mobile.js]: My mistake, I thought there were SVGs for this.
